### PR TITLE
fix(core): user is not redirected to Login page after requesting password reset link

### PR DESCRIPTION
**Pull request by bc-yevhenii-buliuk**
## What/Why?
This PR fixes the flow of redirecting users to the Login Page after changing the password

## Testing
locally

**before:**

https://github.com/bigcommerce/catalyst/assets/82589781/636fcd57-79ff-42a3-95b4-f7abbe559251

https://github.com/bigcommerce/catalyst/assets/82589781/8212c532-90e2-42bf-8e09-97735e668c06

**after:**

https://github.com/bigcommerce/catalyst/assets/82589781/f46f3a41-1a8f-4b0f-a907-2a0b384ac425

https://github.com/bigcommerce/catalyst/assets/82589781/475c7df3-ec6e-484d-9189-c67b742e3678

### CODE REVIEW

**Comment by bc-yevhenii-buliuk** [comment]
To redirect to the Login, I used the context that was created for the Account, so as not to create a separate context for the Login. Guys, do you think this approach is acceptable in this case? If so, should we move this context out of the Account directory and rename it to something more generic to use for both Account and Login? thanks!

**Review by chanceaclark — APPROVED** [review]

**Comment by chanceaclark** [comment]
Some functional tests would be good with this PR as well 🙏

**Review by yurytut1993 — APPROVED** [review]

File: core/messages/en.json [hunk 1]
```diff
@@ -286,7 +286,8 @@
       "confirmPasswordValidationMessage": "Entered passwords are mismatched. Please try again.",
       "newPasswordValidationMessage": "New password must be different from the current password or/and match confirm password.",
       "notEmptyMessage": "Field should not be empty",
-      "successMessage": "Password has been updated successfully!"
+      "successMessage": "Password has been updated successfully!",
+      "confirmChangePassword": "Your password has been successfully updated."
```
Comment (bc-alexsaiannyi) [inline core/messages/en.json:290]: Do we need both translations?

Comment (bc-yevhenii-buliuk) [reply core/messages/en.json]: @bc-alexsaiannyi I've double-checked this message in Figma and fixed it

**Review by bc-alexsaiannyi — APPROVED** [review]
lgtm

File: core/messages/en.json (hunk 1 above)
Comment (bc-alexsaiannyi) [reply core/messages/en.json]: thanks!

**Pull request merged**